package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "antwerp",
```

Comment:
I like it, but why `antwerp`?

Reply:
Naming things is hard.  I started out with pubtools but then I thought I'd change it to pubtool since this is really only one tool, but that's already taken.  I started thinking about pubs I've been in in videos games and there was one in Hero's Quest I (aka Quest for Glory I) where you have to say a secret password to the goon guarding the entrance to the thieves' guild.  One of the passwords was `antwerp`.  I'm open to better names as this name is not very descriptive and only tangentially related to the task that the tool performs.

Reply:
Also, `antwerp` is the name of this creature in the game:
![6c45017fb313c6dd0445afbecab640a7 gif](https://user-images.githubusercontent.com/1044413/41513931-9c50b6e4-7270-11e8-812b-02c6fa6d1286.jpeg)

In my journey through Benelux in the spring I visited Antwerp the city which has this really old wooden escalator which leads to a tunnel that you can walk, bike, motorbike through.  Unsurprisingly it smells really woody.  If you're ever in Antwerp it's a must see... along with the [MAS](https://en.wikipedia.org/wiki/Museum_aan_de_Stroom).